test/known_problems/should_fail/intersection.erl

```diff
@@ -1,9 +1,6 @@
 -module(intersection).
--export([j/1]).
+-export([i/2]).
```

Comment:
`i/2` shouldn't fail, should it? It has no type errors IMO. Only `j/1` was supposed to fail here.

Reply:
![image](https://user-images.githubusercontent.com/152020/64995092-455d8780-d8d2-11e9-9ba9-d2d812019aa6.png)

Yeah, this file should be removed now.